kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaTopicManager.java

```diff
@@ -387,4 +396,33 @@ public void deReference(String topicName) {
         }
     }
 
+    public CompletableFuture<Consumer> getGroupConsumers(String groupId, TopicPartition kafkaPartition) {
+        // make sure internal consumer existed
+        CompletableFuture<Consumer> consumerFuture = new CompletableFuture<>();
+        if (!requestHandler.getGroupCoordinator()
+                .getOffsetAcker().getConsumer(groupId, kafkaPartition).isDone()) {
+            log.warn("not get consumer for group {} this time", groupId);
+            consumerFuture.complete(null);
+            return consumerFuture;
+        }
```

Comment:
After debugging for a while, the `GlobalKTableTests` failures may be caused by this code block. If I commented line 402 to 407, the tests would pass.

Could you take a look?

Reply:
it will be stuck in `createConsumer`, I think there is not a problem.
![image](https://user-images.githubusercontent.com/18047329/106852666-64d99800-66f3-11eb-92a7-a1c332bc712b.png)

Reply:
Yeah it's strange here. I'll dig deeper into this problem when I'm free.